tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{26,27,32,33,34}
+envlist = py{27,32,33,34,35}
```

Comment:
Yeah, I know that will ruffle some feathers but Python 2.6.9 was released 2 years ago, and notes with that release state:

> With the 2.6.9 release, and five years after its first release, the Python 2.6 series is now officially retired. All official maintenance for Python 2.6, including security patches, has ended. For ongoing maintenance releases, please see the Python 2.7 series. 

https://www.python.org/download/releases/2.6.9/